src/accountsdb/db.zig

```diff
@@ -275,25 +275,26 @@ pub const AccountsDB = struct {
         /// needs to be a thread-safe allocator
         allocator: std.mem.Allocator,
         /// Must have been allocated with `self.allocator`.
-        combined_manifest: *FullAndIncrementalManifest,
+        combined_manifest: FullAndIncrementalManifest,
         n_threads: u32,
         validate: bool,
         accounts_per_file_estimate: u64,
         should_fastload: bool,
         save_index: bool,
     ) !SnapshotManifest {
-        const snapshot_fields = try combined_manifest.collapse(self.allocator);
+        const snapshot_manifest = try combined_manifest.collapse(self.allocator);
```

Comment:
should we rename this to `collapsed_manifest` ?

Reply:
b25e2e31dba827f6809c790866d4c2c3cc288920